CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+cmake_minimum_required(VERSION 3.13)
+
+project(ble_sim C CXX ASM)
+
+set(CMAKE_C_STANDARD 11)
```

Comment:
Do not need to set C ASM.
We can add it later, if we need to do so.

Reply:
I also removed C standard from CMakeLists.txt.